### ISSUE TITLE
Missing Races from Power County ID 

### ISSUE THREAD

**Issue by rsolo19** [MEDSL/2022-elections-official#19]
Hello! I noticed there may be some missing races from the Power County, ID results. The below query shows that we have votes for STATE SENATE, ATTORNEY GENERAL, COUNTYWIDE VOTING STATISTICS, and then 0 votes for Governor and Secretary of State. But looking [online](https://www.livevoterturnout.com/ENR/idaho/124/77/en/Index_124.html?v=20221125105447) at the Power County results from 2022, I think we'd expect results from other contests too, such as US Senate, US House, and > 0 votes for Governor.

```
select office, sum(votes)
from `src_medsl__id_2022`
where state_code = 'ID'
and county_name = 'POWER'
and election_year = '2022'
group by 1 
```

As always thanks for all of your work on this :) Let me know if I'm missing something!!

**Comment by sbaltzmit** [comment]
Good catch, I think you're right. I just checked the precinct data that we downloaded from the county, and it looks like several pages of the Excel spreadsheets have broken references ("#REF!") instead of the vote totals. I'll add a note in the readme and I'll see if I can find a better data source, and please let me know if you have already found a way to obtain the correct precinct-level results for those contests. Thanks!
